src/main/java/com/iridium/iridiumskyblock/database/Island.java

```diff
@@ -72,6 +72,10 @@
 
     // Cache
     private Integer size;
+    
+    // Cache Position Island Reset every 5 seconds
+    private final Cache<Location> locationCachePos1 = new Cache<>(5000);
```

Comment:
I don't know if `pos1Cache` or `pos1LocationCache` would be better, this sounds a bit weird, but you don't have to change it if you don't want to